ui/shared/components/panel/family.jsx

```diff
@@ -46,6 +51,56 @@ const mapSampleDispatchToProps = (state, ownProps) => ({
 
 const FirstSample = connect(mapSampleDispatchToProps)(BaseFirstSample)
 
+const AnalystEmailDropdown = ({ load, loading, usersByUsername, onChange, value, ...props }) =>
+  <DataLoader load={load} loading={false} content>
+    <AddableSelect
+      loading={loading}
+      additionLabel="Assigned Analyst: "
+      onChange={val => onChange(usersByUsername[val] || { email: val })}
+      value={value.username || value.email}
+      {...props}
+    />
+  </DataLoader>
+
+AnalystEmailDropdown.propTypes = {
+  load: PropTypes.func,
+  loading: PropTypes.bool,
+  usersByUsername: PropTypes.object,
+  onChange: PropTypes.func,
+  value: PropTypes.any,
+}
+
+const mapDropdownStateToProps = state => ({
+  loading: getUserOptionsIsLoading(state),
+  options: getUserOptions(state),
+  usersByUsername: getUsersByUsername(state),
+})
+
+const mapDropdownDispatchToProps = {
+  load: loadAnalystOptions,
+}
+
+AnalystEmailDropdown.propTypes = {
+  load: PropTypes.func,
+  loading: PropTypes.bool,
+  usersByUsername: PropTypes.object,
+  onChange: PropTypes.func,
+  value: PropTypes.any,
+}
+
+const EDIT_FIELDS = [
+  {
+    name: 'user',
+    label: 'Email',
+    component: connect(mapDropdownStateToProps, mapDropdownDispatchToProps)(AnalystEmailDropdown),
+    validate: value => (
```

Comment:
I understand why you added this validation, and generally you would be right that its a good idea. However, when you are choosing values from a dropdown you generally don't need to validate them. For example, it would be really weird if there was an invalid email address in that dropdown. But as a user, it would be really confusing if you chose an un-editable option from a dropdown and then were told it was invalid